traffic_ops/experimental/server/api/deliveryservices.go

```diff
@@ -179,6 +179,7 @@ func postDeliveryservice(payload []byte, db *sqlx.DB) (interface{}, error) {
 	sqlString += ",dns_bypass_cname"
 	sqlString += ",tr_request_headers"
 	sqlString += ",created_at"
+	sqlString += ",anonymous_blocking_enabled"
```

Comment:
This app isn't used, it's an old experimental version that needs to be removed from the repo.